### PR TITLE
Separated UUID and ScaledCount2

### PR DESCRIPTION
**Pull request by tvincentNuoDB**
Moving ScaleCount 1 & 2 out of UUID seems to have fixed some bugs with large NUMERIC and DECIMAL data types. Fixes [Issue 15](https://github.com/nuodb/nuodb-python/issues/25) and JIRA [DB-10835](http://tools/jira/browse/DB-10835)

### CODE REVIEW

**Comment by tvincentNuoDB** [comment]
With some testing of ScaledCount2 in the system.connections, it is able to fetch execid properly without loss of precision. 
```python
>>> r.execute("select execid from system.connections")
>>> r.fetchall()
[(Decimal('18446744099479355394'),), (0,)]
```

**Comment by tvincentNuoDB** [comment]
I'll revise with this change, I was unsure how the value was to be encoded, now know it's a tuple

File: pynuodb/encodedsession.py [hunk 1]
```diff
@@ -749,9 +765,17 @@ def getValue(self):
...
         
         # get uuid type
-        elif typeCode in [protocol.UUID, protocol.SCALEDCOUNT1, protocol.SCALEDCOUNT2]:
+        elif typeCode is protocol.UUID:
             return self.getUUID()
-        
+
+        # get Scaled Count 1 type
+        elif typeCode is protocol.SCALEDCOUNT1:
+            return self.getScaledCount1()
```
Comment (mjmichaels) [inline pynuodb/encodedsession.py]: Where is getScaledCount1() defined. I am not seeing it here?

**Comment by tvincentNuoDB** [comment]
My bad, pulled the method but, forgot to drop the reference in getValue

File: pynuodb/encodedsession.py [hunk 2]
```diff
@@ -386,11 +386,13 @@ def putInt(self, value, isMessageId=False):
         self.__output += packed
         return self
 
+    #Does not preserve E notation
     def putScaledInt(self, value):
         """
         Appends a Scaled Integer value to the message.
         @type value decimal.Decimal
         """
+        value += 0
```
Comment (mjmichaels) [inline pynuodb/encodedsession.py]: What is the point of this?

File: pynuodb/encodedsession.py [hunk 3]
```diff
@@ -675,7 +677,8 @@ def getClob(self):
             return self._takeBytes(strLength)
 
         raise DataError('Not a clob')
-    
+
+    #Does not preserve E notation
```
Comment (mjmichaels) [inline pynuodb/encodedsession.py]: Should this comment be moved to getScaledCount2()?

File: pynuodb/encodedsession.py [hunk 4]
```diff
@@ -749,9 +767,13 @@ def getValue(self):
             return self.getBoolean()
         
         # get uuid type
-        elif typeCode in [protocol.UUID, protocol.SCALEDCOUNT1, protocol.SCALEDCOUNT2]:
+        elif typeCode is protocol.UUID:
             return self.getUUID()
-        
+
```
Comment (mjmichaels) [inline pynuodb/encodedsession.py]: I thought we agreed that protocol.SCALEDCOUNT1 would still do a pass as it did before?

File: pynuodb/encodedsession.py (hunk 3 above)
Comment (tvincentNuoDB) [inline pynuodb/encodedsession.py]: Whoops, meant to be on ScaledInt actually, I'll move it

File: pynuodb/encodedsession.py (hunk 4 above)
Comment (tvincentNuoDB) [inline pynuodb/encodedsession.py]: Okay, I will update this and push

File: pynuodb/encodedsession.py (hunk 2 above)
Comment (tvincentNuoDB) [inline pynuodb/encodedsession.py]: This will drop any e notation that the decimal may hold

Comment (mjmichaels) [inline pynuodb/encodedsession.py]: That would be a useful comment.

**Pull request merged**